components/api.ts

```diff
@@ -0,0 +1,37 @@
+interface QueryParams {
+  page: number;
+  pageSize: number;
+  orderBy: string;
+  keyWord?: string;
+}
+
+export const getPost = async ({
+  page,
+  pageSize,
+  orderBy,
+  keyWord,
+}: QueryParams) => {
+  const response = await fetch(
+    `https://panda-market-api.vercel.app/articles?page=${page}&pageSize=${pageSize}&orderBy=${orderBy}${
+      !keyWord ? "" : `&keyword=${keyWord}`
+    }`
+  );
+  const body = await response.json();
+  return body;
```

Comment:
body를 사용하거나 변환할 것이 아니면 
```suggestion
  return await response.json();
```
바로 리턴해줘도 좋을 것 같습니다~